.github/workflows/rust.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Rust CI"
+on:
+  workflow_call:
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare dependencies
+        run: |
+          rustup install stable
```

Comment:
We *might* want to go for a different target here. Stable is good, but the default target will be x86_64-unknown-linux-gnu, but I think we want x86_64-unknown-linux-musl for broader compatibility, as that'll be fully statically linked then, instead of depending on a specific glibc abi.

Reply:
I was thinking about probably adding a matrix strategy for multiple targets later on, would you consider that important for an initial merge already?